shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -435,12 +431,18 @@ - (BOOL)createShell:(NSString*)entrypoint libraryURI:(NSString*)libraryURI {
     // TODO(amirh/chinmaygarde): remove this, and dynamically change the thread configuration.
     // https://github.com/flutter/flutter/issues/23975
 
+    _threadHost = {threadLabel.UTF8String,  // label
```

Comment:
That the GPU thread will no longer be created when there are platform views in the scene is an artifact of a previous implementation. This has been fixed https://github.com/flutter/engine/pull/16068 and will cause issues with that line of work.